app/Controllers/CopyController.php

```
@@ -114,12 +111,9 @@ public function updateCopy(Request $request, Response $response, mysqli $db, int
      */
     public function deleteCopy(Request $request, Response $response, mysqli $db, int $copyId): Response
     {
-        $data = (array)$request->getParsedBody();
+        $data = (array) $request->getParsedBody();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$data'. (undefined)

(UnusedLocalVariable)